package.json

```diff
@@ -120,5 +122,41 @@
       }
     },
     "helpClass": "./src/services/twilio-help/custom-help"
+  },
+  "release": {
+    "branches": [
+      "main",
+      {
+        "name": "github_action_changes",
+        "prerelease": "rc"
+      }
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      [
+        "@semantic-release/changelog",
+        {
+          "changelogFile": "CHANGES.md"
+        }
+      ],
+      [
+        "@semantic-release/npm",
+        {
+          "npmPublish": false
```

Comment:
Need to mark `true` ? Is it possible to push these configs in the github actions yml file itself instead of marking in `package.json` ?

Reply:
We will work on npm publish as part of [this](https://issues.corp.twilio.com/browse/DII-109) story. 
Regarding the configurations, we can have these configurations in [three ways](https://semantic-release.gitbook.io/semantic-release/usage/configuration) but can't include in github actions yml file:
1. .releaserc file, written in YAML or JSON, with optional extensions: .yaml/.yml/.json/.js
2. release.config.js file that exports an object
3. release key in the project's package.json file
So I thought to add plugins in `package.json` as we added oclif plugins also here. Please let me know if you think do other way.